lib/scanner.js

```diff
@@ -171,6 +172,20 @@ function scanMainQR (cryptoCode, callback) {
   })
 }
 
+function scanPK (callback) {
+  scanQR(function (err, result) {
+    if (err) return callback(err)
+    if (!result) return callback(null, null)
+
+    var resultStr = result.toString()
+    try {
+      callback(null, resultStr)
```

Comment:
Why do we want to ``try`` the callback? Shouldn't we be ``try``ing ``toString``?